### PR TITLE
delete image reference validating info

### PR DESCRIPTION
**Pull request by zhuguihua**
delete image reference validating info.

### CODE REVIEW

**Comment by zhuguihua** [comment]
I signed it

**Comment by zhuguihua** [comment]
I signed it

**Comment by zhuguihua** [comment]
I signed it

**Comment by jeefy** [comment]
I feel like this would make dashboard's logs unnecessarily chatty.

Additionally, if `ValidateImageReference` returns an error it already gets handled.

What value are you trying to add, or what problem are you trying to solve? Thanks in advance!

**Comment by zhuguihua** [comment]
I'd like to know the reason in log when validating image reference failed.

File: src/app/backend/validation/validateimagereference.go [hunk 1]
```diff
@@ -41,7 +41,10 @@ func ValidateImageReference(spec *ImageReferenceValiditySpec) (*ImageReferenceVa
 	s := spec.Reference
 	_, err := reference.ParseNamed(s)
 	if err != nil {
+		log.Printf("Validation result for %s image reference is false, reason: %v", spec.Reference, err)
 		return &ImageReferenceValidity{Valid: false, Reason: err.Error()}, nil
```
Comment (shu-mutou) [inline src/app/backend/validation/validateimagereference.go]: I'm not sure about subsequent handling ( https://github.com/kubernetes/dashboard/blob/664f533db09fc94b158536b5d6cc75fcf99a7283/src/app/backend/handler/apihandler.go#L1054-L1058 ), 
however should the "nil" in L45 and L49 be err object to pass it to caller first?

**Comment by zhuguihua** [comment]
When image name was changed, this api would be called. Maybe it is no necessary to show validating info, just only show validation result?

**Comment by maciaszczykm** [comment]
@zhuguihua Still too much info in the logs IMO.

**Comment by zhuguihua** [comment]
@maciaszczykm I go to try this api again, there is no need to show the validation info indeed. And now I think the validating info can also be deleted.

**Comment by jeefy** [comment]
Your tests are failing because `validateimagereference.go` is still importing the `log` library and it no longer needs to. 

Please remove that import and push your changes. The tests should pass and I can lgtm it. 

Thanks!

**Comment by zhuguihua** [comment]
@jeefy  Thanks, updated.

**Pull request merged**